CondTools/HLT/src/AlCaRecoTriggerBitsRcdUpdate.cc

```diff
@@ -179,28 +171,17 @@ bool AlCaRecoTriggerBitsRcdUpdate::addTriggerLists(const std::vector<edm::Parame
 }
 
 ///////////////////////////////////////////////////////////////////////
-void AlCaRecoTriggerBitsRcdUpdate::writeBitsToDB(AlCaRecoTriggerBits *bitsToWrite) const {
+void AlCaRecoTriggerBitsRcdUpdate::writeBitsToDB(const std::unique_ptr<AlCaRecoTriggerBits> &bitsToWrite) const {
```

Comment:
```suggestion
void AlCaRecoTriggerBitsRcdUpdate::writeBitsToDB(const AlCaRecoTriggerBits &bitsToWrite) const {
```
Here I don't see the reason to pass the smart-ptr by ref rather the (const ref of the) object itself.

Reply:
Yes very good point, I will add this in my next commit as well.